### PR TITLE
[TS] Content: Add support for Firestorm Assets and Music

### PR DESCRIPTION
**Pull request by LipkeGu**
This PR is basically a PR of #11787, #12047, #12048 in one PR to fix the mirror problems
and enables the Installation and Download of "stripped down" Firestorm Assets and Music.

Most work and thanks goes to @Mailaender

### CODE REVIEW

File: mods/ts/installer/downloads.yaml
Comment (pchote) [inline mods/ts/installer/downloads.yaml]: Please keep the displayed names as they are.  The naming (a) makes it clear that the original games were released as freeware, and (b) keeps consistency between the mods.

**Review by pchote — CHANGES_REQUESTED** [review]

File: mods/ts/installer/downloads.yaml
Comment (LipkeGu) [inline mods/ts/installer/downloads.yaml]: updated

File: mods/ts/installer/downloads.yaml
Comment (pchote) [inline mods/ts/installer/downloads.yaml]: Please extract out just the unique files from inside these mixes too.

**Review by pchote — CHANGES_REQUESTED** [review]

File: mods/ts/installer/downloads.yaml
Comment (LipkeGu) [inline mods/ts/installer/downloads.yaml]: updated

**Comment by LipkeGu** [comment]
fona

**Comment by LipkeGu** [comment]
updated

**Comment by LipkeGu** [comment]
Can anyone share the tibsun.mix and expand01.mix from orign-installations?

**Comment by LipkeGu** [comment]
@Mailaender I gave you Write Access to this Branch so you can work with this entire Branch... Reaper or Jugglernaut.

**Comment by Mailaender** [comment]
Thanks for taking this task over and for the invitation, but I won't just interfere here.

**Comment by LipkeGu** [comment]
updated

File: mods/ts/installer/origin.yaml
Comment (pchote) [inline mods/ts/installer/origin.yaml]: You've accidentally removed the end-of-file newline here.

File: mods/ts/installer/tibsun.yaml
Comment (pchote) [inline mods/ts/installer/tibsun.yaml]: This change can be dropped too.

File: mods/ts/installer/downloads.yaml
Comment (pchote) [inline mods/ts/installer/downloads.yaml]: Can you please add these to https://github.com/OpenRA/OpenRAWeb/pull/316?  You'll need to create a new mirror txt file. Once that is merged we can set these properly.

**Review by pchote — COMMENTED** [review]
This all looks correct, but I haven't verified whether this includes _all_ the unique firestorm content.  @ABrandau can you double check this?

Please see the nits below.

**Comment by pchote** [comment]
:+1: after the nits have been fixed (assuming no other nits work their way in as part of that).

**Comment by LipkeGu** [comment]
Updated and switched to MirrorList for the time until the mirrors will be updated the links are stored here

firestorm files: http://81.169.246.181/openra/ts/ts-expand.zip
firestorm Music: http://81.169.246.181/openra/ts/ts-fsmusic.zip

**Comment by ABrandau** [comment]
Dawnbridges loads fine but the quick install function didn't work for me, seems it downloaded some files and then discarded them, created a loop, I had to use the advanced install option to get the Firestorm data.

**Comment by Mailaender** [comment]
> Most work and thanks goes to @Mailaender

Actually most work and thanks goes to @ABrandau: https://github.com/ABrandau/OpenRA/tree/FS-Files-Inclussion

> Updated and switched to MirrorList for the time until the mirrors will be updated

https://github.com/OpenRA/OpenRAWeb/pull/316 has been merged. Hope this helps.

**Comment by Iamgoofball** [comment]
I've got the Origin collection US edition if you still need those mixes

**Comment by Mailaender** [comment]
The purpose of this pull request is not to generate those mix files, but find a procedure so anyone can do it himself with a simple graphical user interface. You can help test this patch and give your :+1: if it works for you.

**Comment by LipkeGu** [comment]
>Dawnbridges loads fine but the quick install function didn't work for me, seems it downloaded some files >and then discarded them, created a loop, I had to use the advanced install option to get the Firestorm >data.

it is currently not validated that the orign US mixes are valid, we need hashes can you send me the sha1 hash of expand01 or the file expand01 file itself?

the files gets extracted not copied

File: mods/ts/sequences/trees.yaml
Comment (reaperrr) [inline mods/ts/sequences/trees.yaml]: Why not use bigblue3?

File: mods/ts/installer/tibsun.yaml
Comment (reaperrr) [inline mods/ts/installer/tibsun.yaml]: @LipkeGu You haven't adressed this yet.

File: mods/ts/mod.yaml
Comment (pchote) [inline mods/ts/mod.yaml]: You need to package a new quickinstall zip that contains all the essential content (base + firestorm, without music), and then reference it here.

**Review by pchote — CHANGES_REQUESTED** [review]

File: mods/ts/mod.yaml
Comment (pchote) [inline mods/ts/mod.yaml]: This is why the quick install button doesn't work.

**Comment by pchote** [comment]
Any progress here @LipkeGu?

**Comment by docholiday** [comment]
Hi there, the firestorm files and music seem to be down (I always get a timeout). Could you please make them available again?

**Comment by reaperrr** [comment]
Superseded by #12429.

**Pull request closed without merging**
